### PR TITLE
Caf updates

### PR DESCRIPTION
**Pull request by RobertFewins-Kalb**
Two updates...
(1) Added Addendum to the droplist of CAF types
(2) Added TIKL to prompt the worker to APPL an incomplete CAF reinstate for CASH/SNAP

### CODE REVIEW

File: Script Files/NOTES/NOTES - CAF.vbs [hunk 1]
```diff
@@ -465,6 +465,21 @@ If client_delay_TIKL_checkbox = checked then
 	transmit
 	PF3
 End if
+'----Here's the new bit to TIKL to APPL the CAF for CAF_datestamp if the CL fails to complete the CASH/SNAP reinstate and then TIKL again for DateAdd("D", 30, CAF_datestamp) to evaluate for possible denial.
+'----IF the DatePart("M", CAF_datestamp) = footer_month (DatePart("M", CAF_datestamp) is converted to footer_comparo_month for the sake of comparison) and the CAF_status <> "Approved" and CAF_type is a recertification AND cash or snap is checked, then 
+'---------the script generates a TIKL.
+footer_comparo_month = DatePart("M", CAF_datestamp)
```
Comment (theVKC) [inline Script Files/NOTES/NOTES - CAF.vbs]: comparo? @RobertFewins-Kalb

File: Script Files/NOTES/NOTES - CAF.vbs [hunk 2]
```diff
@@ -60,7 +60,7 @@ BeginDialog case_number_dialog, 0, 0, 181, 120, "Case number dialog"
   CheckBox 50, 60, 30, 10, "HC", HC_checkbox
   CheckBox 90, 60, 35, 10, "SNAP", SNAP_checkbox
   CheckBox 135, 60, 35, 10, "EMER", EMER_checkbox
-  DropListBox 70, 80, 75, 15, "Intake"+chr(9)+"Reapplication"+chr(9)+"Recertification"+chr(9)+"Add program", CAF_type
+  DropListBox 70, 80, 75, 15, "Intake"+chr(9)+"Reapplication"+chr(9)+"Recertification"+chr(9)+"Add program"+chr(9)+"Addendum", CAF_type
```
Comment (theVKC) [inline Script Files/NOTES/NOTES - CAF.vbs]: Does this case note @RobertFewins-Kalb?

**Comment by theVKC** [comment]
@RobertFewins-Kalb, can you fix camparo? Huh? Huh? Huh?

**Comment by RobertFewins-Kalb** [comment]
Updated footer_comparison_month.

And the case note header includes "Addendum"

**Comment by theVKC** [comment]
Excellent! This one closes #89.

**Pull request merged**